packages/firestore/test/unit/local/test_remote_document_cache.ts

```diff
@@ -67,13 +68,42 @@ export class TestRemoteDocumentCache {
     );
   }
 
-  getNextDocumentChanges(): Promise<MaybeDocumentMap> {
+  getNewDocumentChanges(): Promise<MaybeDocumentMap> {
     return this.persistence.runTransaction(
-      'getNextDocumentChanges',
+      'getNewDocumentChanges',
       'readonly',
       txn => {
         return this.cache.getNewDocumentChanges(txn);
       }
     );
   }
+
+  resetLastProcessDocumentChange(): Promise<void> {
```

Comment:
I'm hoping this method goes away, but if not you're missing 'ed' here and below.

Reply:
And it's gone!